steps/cordova-config/0.0.1/step.yml

```diff
@@ -0,0 +1,64 @@
+title: cordova-config
+summary: |
+  Manipulate cordova config.xml main attributes
+description: |
+  Manipulate cordova config.xml main attributes
+website: https://github.com/platanus/bitrise-step-cordova-config
+source_code_url: https://github.com/platanus/bitrise-step-cordova-config
+support_url: https://github.com/platanus/bitrise-step-cordova-config/issues
+published_at: 2017-12-06T17:18:19.402156-05:00
+source:
+  git: https://github.com/platanus/bitrise-step-cordova-config.git
+  commit: 3f05242339b51977f5bfa50e4f6e16b5b0069102
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- cordova
+- ionic
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- name: $CORDOVA_APP_NAME
+  opts:
+    is_expand: true
+    is_required: false
+    summary: Modify the element `<name></name>` in the config.xml file
+    title: Cordova application name
+- id: $CORDOVA_ID
+  opts:
+    is_expand: true
+    is_required: false
+    summary: Modify the attribute `id` the config.xml file
+    title: Cordova ID
+- android_versionCode: $CORDOVA_ANDROID_VERSIONCODE
+  opts:
+    is_expand: true
+    is_required: false
+    summary: Modify the attribute `android-versionCode` in the config.xml file
+    title: Android version code
+- android_packageName: $CORDOVA_ANDROID_PACKAGENAME
+  opts:
+    is_expand: true
+    is_required: false
+    summary: Modify the attribute `android-packageName` in the config.xml file
+    title: Android package name
+- ios_CFBundleVersion: $CORDOVA_IOS_BUNDLEVERSION
```

Comment:
please use snake_case input keys, like: `ios_bundle_version`

Reply:
Yeah, you're right. I was trying to stick to the original attribute format, but... is ugly. And people don't need to really know whats happening underneath.